### PR TITLE
dev-libs/ivykis: make static libs optional

### PR DESCRIPTION
**Pull request by hydrapolic**
https://github.com/gentoo/gentoo/pull/8799#issuecomment-396895324

### CODE REVIEW

File: dev-libs/ivykis/ivykis-0.42.3-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+src_configure() {
+	econf $(use_enable static-libs static)
+}
+
+src_install() {
+	default
+
+	if ! use static-libs; then
+		find "${ED}" -name "*.la" -delete || die
```
Comment (mgorny) [inline dev-libs/ivykis/ivykis-0.42.3-r1.ebuild]: Do you really need the .la file when .pc file is supplied?

Comment (mgorny) [reply dev-libs/ivykis/ivykis-0.42.3-r1.ebuild]: (and there are no dependencies apparently)

Comment (hydrapolic) [reply dev-libs/ivykis/ivykis-0.42.3-r1.ebuild]: Yes, by reading https://wiki.gentoo.org/wiki/Project:Quality_Assurance/Handling_Libtool_Archives it really seems "*.la" can go, I'll adjust, thanks.

**Comment by hydrapolic** [comment]
@mgorny, updated, thanks

**Comment by mgorny** [comment]
Thanks!

**Pull request closed without merging**